tests/x11regressions/gnomecase/gnome_default_applications.pm

```diff
@@ -0,0 +1,71 @@
+# Gnome tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+
+# Case 1503753: Gnome - Some types of files should be opened by corresponding applications
+
+sub run() {
+    my $self = shift;
+
+    # Prepare test files
+    x11_start_program("mkdir /home/$username/gnometest");
```

Comment:
x11_start_program isn't really made for mkdir calls. Just start a terminal using it and then use assert_script_run in that xterm and close the terminal afterwards.